content/blog/2022/04/securing-kubewarden-policies.md

```diff
@@ -0,0 +1,256 @@
+```
+
+The end result is the same, a signature added, as a new image layer of a special OCI object created and managed by Sigstore. You can see those signatures as added
+[layers](https://github.com/kubewarden/user-group-psp-policy/pkgs/container/policies%2Fuser-group-psp/15759776?tag=v0.2.0),
+with [`sha256-<sha>.sig` in the
+repo](https://github.com/kubewarden/user-group-psp-policy/pkgs/container/policies%2Fuser-group-psp/versions).
+Or better, either using `kwctl pull <policy_url>; kwctl inspect <policy_url>` or
+with tools like [`crane`](https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane.md).
+
+If you want to verify policies locally, you now can use `kwctl verify`:
+```console
+$ kwctl verify --github-owner kubewarden registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.10
+$ echo $?
+0
+```
+
+And when testing policies locally with `kwctl pull` or `kwctl run`, you can also
+enable signature verification by using any verification related flag. For example:
+```console
+$ kwctl pull --github-owner kubewarden registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.10
+$ echo $?
+0
+```
+
+
+All the policies from the Kubewarden team are signed in keyless mode by the
+workers of the CI job. In our case, the workers used in the CI job of Github.
+We don't leave certs around, and they are verifiable by third parties.
+
+### Enforcing signature verification for instantiated Kubewarden policies
+
+You can now configure
+[PolicyServers](https://doc.crds.dev/github.com/kubewarden/kubewarden-controller/policies.kubewarden.io/PolicyServer/v1alpha2)
+to enforce that all policies being run need to be signed. When
+deploying Kubewarden via Helm charts, you can do it so for the default
+PolicyServer installed by `kubewarden-defaults` chart.
+
+For this, the PolicyServers have a new [`spec.VerificationConfig`](https://doc.crds.dev/github.com/kubewarden/kubewarden-controller/policies.kubewarden.io/PolicyServer/v1alpha2#spec-verificationConfig) argument.
+Here, you can put the name of a ConfigMap containing a "verification config", to
+specify the needed signatures.
+
+You can obtain a default verification config for policies from the Kubewarden
+team with:
+
+```yaml
+$ kwctl scaffold verification-config
+# Default Kubewarden verification config
+#
+# With this config, the only valid policies are those signed by Kubewarden
+# infrastructure.
+#
+# This config can be saved to its default location (for this OS) with:
+#   kwctl scaffold verification-config > /home/youruser/.config/kubewarden/verification-config.yml
+#
+# Providing a config in the default location enables Sigstore verification.
+# See https://docs.kubewarden.io for more Sigstore verification options.
+---
+apiVersion: v1
+allOf:
+  - kind: githubAction
+    owner: kubewarden
+    repo: ~
+    annotations: ~
+anyOf: ~
+```
+
+The verification config format has several niceties, [see its reference docs](TODO point to VerificationConfig reference in docs).
```

Comment:
REMINDER: put the link

Reply:
Done in https://github.com/kubewarden/kubewarden.io/pull/82/commits/0093c152ae5b278f4cc41cc7c690f3ea7745d44c.